chain.schema.json

```diff
@@ -502,7 +528,6 @@
       }
     }
   },
-  "additionalProperties": false,
```

Comment:
why removing this restriction?

Reply:
it is duplicated, looking at line 505 and line 613 of the most recent commit
https://github.com/cosmos/chain-registry/blob/3caf657a917a32ec478b98932e898beea990e2f5/chain.schema.json